pfcpiface/bess.go

```diff
@@ -37,10 +37,8 @@ const (
 	SessQerLookup = "sessionQERLookup"
 	// PreQosFlowMeasure: Pre QoS measurement module name.
 	PreQosFlowMeasure = "preQosFlowMeasure"
-	// PostDlQosFlowMeasure: Post QoS measurement downlink module name.
-	PostDlQosFlowMeasure = "postDLQosFlowMeasure"
-	// PostUlQosFlowMeasure: Post QoS measurement uplink module name.
-	PostUlQosFlowMeasure = "postULQosFlowMeasure"
+	// PostQosFlowMeasure: Post QoS measurement module name.
+	PostQosFlowMeasure = "postQosFlowMeasure"
```

Comment:
This breaks flow measure on the bess pipeline. Since it's not really needed there anyway, should we move the entire stat read code to `aether.go`?

Reply:
I don't have a strong opinion on that. I'm not much familiar with Aether requirements on flow measurements. I'll let you or @ccascone decide.

Reply:
I don't have a strong opinion on that. I'm not much familiar with Aether requirements on flow measurements. I'll let you or @ccascone decide.

Reply:
Here's the short summary: Aether wants per-flow stats via Prometheus. There is a performance overhead with tracking every flow, which is undesirable for some deployments with the current upf pipeline. Hence we made it an optional config choice. For the new pipeline we don't have to consider these use cases and can just permanently enable it, making the code easier.